Rocket-Nozzle-Design-and-Analysis/Rocket_Nozzle_Design_and_Analysis.tex

```diff
@@ -0,0 +1,235 @@
+
+\section{Implementation}\label{sec:implementation}
+The project will start with a brief eduation of the rocket equations. The team will go through each one so everyone on the team is acquainted with them before we dive into the
+nozzle design. Then the team will start with the nozzle design. Once the nozzle design is complete the team will design the test stand that can adequately contain the
+propulsion system. Then the team will manufacture the nozzle and test stand. After testing, if time permits, the team will use a schlieren lensing setup to
+get a very good look at the fluid flowing out of the nozzle.
+
+\subsection{Deliverables}\label{subsec:deliverables}
+The physical deliverables for this project are the test stand and rocket nozzle. The test stand and rocket nozzle can be shown to faculty and other
+bodies that could support SPEX in some fashion. The test stand and nozzle will be stored so it can be demonstrated when necessary, such as an introduction to SPEX
+engineering to students, faculty, or others.
+
+The non-physical deliverables would be a poster that exhibits the work completed and the results of the experiment. There will be a final report that outlines
+the method used for designing the nozzle and manufacturing the nozzle and test stand. The report will be written in the IEEE standard and can be submitted to various
+organizations for funding other outreach opportunities.
+
+\subsection{Milestones}\label{subsec:milestones}
+The milestones for this project are, in chronological order; nozzle design, test stand design, nozzle fabrication, test stand fabrication, testing, and final report.
+The nozzle design and fabrication proceed the test stand because the test stand must be designed to fit the nozzle, the nozzle should not be designed to fit the test stand.
+The following, is a proposed project schedule.
+
+\begin{table}
+  \centering
+  \begin{tabularx}{\columnwidth}{@{}cXc@{}}
+    \multicolumn{2}{ c }{\textbf{Proposed Timeline}} \\ \toprule
+    Week & Description \\ \midrule
+    1 & Team Introductions and project planning \\
+    2 & Nozzle Design \\
+    3 & Nozzle Design \\
+    4 & Nozzle Design \\
+    5 & Nozzle Design \\
+    6 & Test Stand Design \\
+    7 & Test Stand Design \\
+    8 & Test Stand Design \\
+    9 & Nozzle Fabrication \\
+    10 & Test Stand Fabrication \\
+    11 & Test Stand Fabrication \\
+    12 & Order Gas \\
+    13 & Setup experiment \\
+    14 & Test \#1 \\
+    15 & Test \#2 \& Schlieren Lensing \\
+    \bottomrule
+  \end{tabularx}
+\label{tab:proposed-timeline}
+\end{table}
+
+\begin{}
+\section{Externalities}
+  % Things not directly related to the work or outcomes, but related to the project as a whole.
+\subsection{Prerequisite Skills}
+No skills are required. But the recommended skills are as follows; Differential \& Integral Calculus and MATLAB coding. The rocket equations
```

Comment:
What engineering and machining skills are required? Are there clear roles individuals can fall into? How many of each role would be needed at a minimum? Is the project viable with 2 people, 10, 20?